### PR TITLE
[HOTFIX 1] 7.0.0b1

### PR DESCRIPTION
**Pull request by HunterBarclay**
### Fixes
- Scheme selection in Manage Assemblies

### CODE REVIEW

File: fission/src/ui/modals/spawning/ManageAssembliesModal.tsx [hunk 1]
```diff
@@ -1,30 +1,50 @@
...
-                value="Delete"
-                onClick={() => {
-                    World.SceneRenderer.RemoveSceneObject(mira.id)
-                    update()
-                }}
-            />
+            <Label className="text-wrap break-all">{`[${InputSystem.brainIndexSchemeMap.get(brain)?.schemeName ?? "-"}] ${mira.assemblyName}`}</Label>
+            <div className="flex flex-row gap-4">
+                <Button
+                    value="Set Scheme"
```
Comment (Dhruv-0-Arora) [inline fission/src/ui/modals/spawning/ManageAssembliesModal.tsx:46]:
```suggestion
                    value="Edit Scheme"
```

File: fission/src/ui/modals/spawning/ManageAssembliesModal.tsx
Comment (Dhruv-0-Arora) [inline fission/src/ui/modals/spawning/ManageAssembliesModal.tsx:1]: When you edit the scheme theres no cancel button and you can't set it to back to ernie so you r kinda forced to change it. can there be a cancel button or at least the current configuration also appearing on the list

File: fission/src/ui/modals/spawning/ManageAssembliesModal.tsx [hunk 2]
```diff
@@ -1,30 +1,50 @@
...
-            <Button
-                value="Delete"
-                onClick={() => {
-                    World.SceneRenderer.RemoveSceneObject(mira.id)
-                    update()
-                }}
-            />
+            <Label className="text-wrap break-all">{`[${InputSystem.brainIndexSchemeMap.get(brain)?.schemeName ?? "-"}] ${mira.assemblyName}`}</Label>
+            <div className="flex flex-row gap-4">
+                <Button
```
Comment (Dhruv-0-Arora) [inline fission/src/ui/modals/spawning/ManageAssembliesModal.tsx:44]: can the button be blue with a pencil icon 🫠

**Review by Dhruv-0-Arora — DISMISSED** [review]

File: fission/src/ui/modals/spawning/ManageAssembliesModal.tsx (hunk 2 above)
Comment (Dhruv-0-Arora) [reply fission/src/ui/modals/spawning/ManageAssembliesModal.tsx:44]: Can the delete button also be red with a trash can next to it

File: fission/src/ui/modals/spawning/ManageAssembliesModal.tsx (hunk 1 above)
Comment (HunterBarclay) [reply fission/src/ui/modals/spawning/ManageAssembliesModal.tsx:46]: This doesn't edit the scheme, it sets it

**Pull request merged**